Managers/ActivityMan.h

```diff
@@ -79,22 +154,33 @@ namespace RTE {
 		void SetStartActivity(Activity *newActivity);
 
 		/// <summary>
-		/// Gets the current Activity in effect. Won't be what has been set by SetStartActivity unless RestartActivity has been called since.
+		/// Loads the "Tutorial Mission" Scene and starts the Tutorial Activity.
 		/// </summary>
-		/// <returns>The current Activity in effect. Will be 0 if no Activity is going.</returns>
-		Activity * GetActivity() const { return m_Activity; }
+		void SetStartTutorialActivity();
 
 		/// <summary>
-		/// Indicates whether the game is currently running or not (not editing, over or paused)
+		/// Loads "Editor Scene" and starts the given editor Activity.
 		/// </summary>
-		/// <returns>Whether the game is running or not.</returns>
-		bool ActivityRunning() const { return m_Activity ? m_Activity->IsRunning() : false; }
+		/// <param name="editorToLaunch">The editor name to put into effect next time ResetActivity is called.</param>
+		void SetStartEditorActivity(const std::string_view &editorToLaunch);
 
 		/// <summary>
-		/// Indicates whether the game is currently paused or not.
+		/// Launch editor Activity specified in command-line argument.
 		/// </summary>
-		/// <returns>Whether the game is paused or not.</returns>
-		bool ActivityPaused() const { return m_Activity ? m_Activity->IsPaused() : true; }
+		/// <returns>Whether a valid editor name was passed in and set to be launched next time ResetActivity is called.</returns>
+		bool SetStartEditorActivitySetToLaunchInto();
```

Comment:
Just following the bad naming of things.
A better point to complain about it would be why does it have `Start` in it when it doesn't actually start anything, and neither any of the other methods except `StartActivity` itself.
`SetStartEditorActivitySetToLaunchInto` sets `m_StartActivity` through `SetStartEditorActivity` calling `SetStartActivity`.
Anyway, fixing the naming is something to do later:tm:.

Reply:
hahaha fugg, you're right, it's garbage naming all the way down.
Yeah, stick in a TODO to rename this shit I guess, and deal with it later.